client.go

```diff
@@ -170,3 +171,92 @@ func Exec(ctx context.Context, cmd string, args ...interface{}) error {
 func Query(ctx context.Context, cmd string, args ...interface{}) Args {
 	return DefaultClient.Query(ctx, cmd, args...)
 }
+
+type MultiError []error
```

Comment:
👍 

Errors are rarely reported as concrete types in Go, instead the `error` interface is used. If we wanted to give access to individual errors we could expose it as an interface that the error may implement, for example something like this:
```go
func (err multiError) Causes() []error {
    return ([]error)(err)
}
...
if causes, ok := err.(interface{ Causes() []error }); ok {
    for _, err := range causes {
        // ...
    }
}
```
That way we have a lose coupling between the error producer and consumer.